### ISSUE TITLE
bug(everything): Something mysterious borked with LuaSnip

### ISSUE THREAD

**Issue by gegnew** [pysan3/neorg-templates#4]
## Bug

Something has gone wrong when executing `Neorg templates <template name>`

I haven't changed my config at all, just updated Packer, so something between `neorg-templates` and `LuaSnip` seems to be broken. It appears that `neorg-templates` is passing a string to Luasnip when it should be passing a object of some sort, but I'm not sure.

## Reproduce

I'm on the most recent versions of Neorg, neorg-templates, and LuaSnip. Execute `Neorg templates .........`

## Template File

Using the minimal "literature" template in the examples.

## Stack Trace

```txt
Error executing Lua callback: vim/_editor.lua:0: nvim_exec2()..BufNewFile Autocommands for "/Users/g/Sync
/vault2/diary/*.norg": Vim:Error executing Lua callback: .../pack/packer/start/LuaSnip/lua/luasnip/nodes/
snippet.lua:70: attempt to index local 'node' (a string value)
stack traceback:
        .../pack/packer/start/LuaSnip/lua/luasnip/nodes/snippet.lua:70: in function 'init_nodes'
        .../pack/packer/start/LuaSnip/lua/luasnip/nodes/snippet.lua:383: in function 'sn'
        ...lua/neorg/modules/external/templates/snippet_handler.lua:61: in function 'build_keywords'
        ...lua/neorg/modules/external/templates/snippet_handler.lua:98: in function 'create_snippet'
        ...lua/neorg/modules/external/templates/snippet_handler.lua:116: in function 'add_snippet_to_luas
nip'
        ...lua/neorg/modules/external/templates/snippet_handler.lua:152: in function 'on_event'
        ...e/nvim/site/pack/packer/start/neorg/lua/neorg/events.lua:149: in function 'broadcast_event'
        ...r/start/neorg/lua/neorg/modules/core/neorgcmd/module.lua:289: in function <...r/start/neorg/lu
a/neorg/modules/core/neorgcmd/module.lua:196>
        [C]: in function 'nvim_exec2'
        vim/_editor.lua: in function 'cmd'
        ...ker/start/neorg/lua/neorg/modules/core/dirman/module.lua:316: in function 'create_file'
        ...er/start/neorg/lua/neorg/modules/core/journal/module.lua:138: in function 'open_diary'
        ...er/start/neorg/lua/neorg/modules/core/journal/module.lua:163: in function 'diary_today'
        ...er/start/neorg/lua/neorg/modules/core/journal/module.lua:494: in function 'on_event'
        ...e/nvim/site/pack/packer/start/neorg/lua/neorg/events.lua:149: in function 'broadcast_event'
        ...r/start/neorg/lua/neorg/modules/core/neorgcmd/module.lua:289: in function <...r/start/neorg/lu
a/neorg/modules/core/neorgcmd/module.lua:196>
stack traceback:
        [C]: in function 'nvim_exec2'
        vim/_editor.lua: in function 'cmd'
        ...ker/start/neorg/lua/neorg/modules/core/dirman/module.lua:316: in function 'create_file'
        ...er/start/neorg/lua/neorg/modules/core/journal/module.lua:138: in function 'open_diary'
        ...er/start/neorg/lua/neorg/modules/core/journal/module.lua:163: in function 'diary_today'
        ...er/start/neorg/lua/neorg/modules/core/journal/module.lua:494: in function 'on_event'
        ...e/nvim/site/pack/packer/start/neorg/lua/neorg/events.lua:149: in function 'broadcast_event'
        ...r/start/neorg/lua/neorg/modules/core/neorgcmd/module.lua:289: in function <...r/start/neorg/lua/neorg/modules/
core/neorgcmd/module.lua:196
```

**Comment by pysan3** [comment]
Thanks for the report @gegnew !

I'll look into it ASAP.

**Comment by pysan3** [comment]
Sorry for the inconvenience @gegnew 

I think I fixed the issue with #5, so could you test the latest main branch?

Also, the `literature.norg` uploaded to my dotfiles was outdated and was using the wrong keywords.

Could you update your templates if you copied from mines? https://github.com/pysan3/dotfiles/blob/main/nvim/templates/norg/literature.norg

**Comment by gegnew** [comment]
Awesome, thanks so much for the fix! Really appreciate this plugin 👍
